lib/init.lua

```diff
@@ -122,35 +114,27 @@ local function ECS_GENERATION_INC(e: i53)
 		local generation = flags % ECS_GENERATION_MASK
 
 		return ECS_COMBINE(id, generation + 1) + flags
+	else
```

Comment:
I prefer not having `else` statements where possible because it is a good pattern to make sure the function always returns.